content/blog/fast-data-processing-pipeline-for-predicting-flight-delays-using-apache-apis-kafka-spark-streaming-and-machine-learning-part-3.md

```diff
@@ -0,0 +1,295 @@
+---
+title: "Fast data processing pipeline for predicting flight delays using Apache
+  APIs: Kafka, Spark Streaming and Machine Learning (part 3)"
+date: 2021-10-11T21:12:59.805Z
+author: Carol McDonald
+authorimage: /img/Avatar1.svg
+tags:
+  - hpe-ezmeral-data-fabric
+  - MapR
+---
+```
+"authorDisplayName": "Carol McDonald",
+"tags": "use-cases",
+"publish": "2018-02-08T12:00:00.000Z"
+```
+
+**Editor’s Note:** MapR products and solutions sold prior to the acquisition of such assets by Hewlett Packard Enterprise Company in 2019 may have older product names and model numbers that differ from current solutions. For information about current offerings, which are now part of HPE Ezmeral Data Fabric, please visit <https://www.hpe.com/us/en/software/data-fabric.html>
```

Comment:
In the previous 2 blog posts, “part” was written with a lower-case “p”. While an upper-case “P” would have been more appropriate, no reason to go back and edit them.  In addition, it’s tied into the URL, I believe. Let’s just keep things consistent and go with the lower-case version in this post. So these should be “3-part” and “(part 3)”.
![image](https://user-images.githubusercontent.com/77496261/137182519-ae87293a-1e28-4917-96ac-d728a04f53ce.png)

Reply:
done